DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -1,18 +1,18 @@
 autoDQM = { 'common': ['@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics+@tau',
                         'PostDQMOffline',
-                        '@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@fed+@tau+dqmFastTimerServiceClient'],
```

Comment:
@fabiocos and @jfernan2 wouldn't be easier to define a new key, like:
'DQMMessageLogger' : ['DQMMessageLoggerSeq', 
                                        'PostDQMOffline',
                                        'DQMMessageLoggerClientSeq']
and then call @DQMMessageLogger in both step1 and step2 ?

Reply:
That was the approach I was testing now... Thanks!